$OpenBSD: patch-samples_gtk-demo_demo-uimanager_lua,v 1.1 2012/08/21 09:36:19 jasper Exp $

gtk-demo: fix uimanager demo, which assigned the same value to all
accelerators

https://github.com/pavouk/lgi/commit/36945b9aa52e68a55ea0c5d6b8e2fcd8a9688c44

--- samples/gtk-demo/demo-uimanager.lua.orig	Tue Aug 21 11:30:35 2012
+++ samples/gtk-demo/demo-uimanager.lua	Tue Aug 21 11:31:01 2012
@@ -17,6 +17,7 @@ local COLOR = { RED = 1, GREEN = 2, BLUE = 3 }
 local SHAPE = { SQUARE = 1, RECTANGLE = 2, OVAL = 3 }
 
 local actions = Gtk.ActionGroup {
+   name = 'Actions',
    Gtk.Action { name = 'FileMenu', label = "_File" },
    Gtk.Action { name = 'PreferencesMenu', label = "_Preferences" },
    Gtk.Action { name = 'ColorMenu', label = "_Color" },
@@ -139,6 +140,7 @@ local window = Gtk.Window {
       Gtk.Box {
 	 orientation = 'VERTICAL',
 	 spacing = 10,
+	 border_width = 10,
 	 Gtk.Button {
 	    id = 'close',
 	    label = "close",
